src/app/admin/admin.component.html

```diff
@@ -138,12 +138,12 @@
         <button
           *ngIf="!submittingGraylistUpdate"
           (click)="updateProfileModerationLevel('graylist')"
-          class="btn btn-secondary fs-15px ml-5px"
+          class="btn btn-dark fs-15px ml-5px"
```

Comment:
make this the same as the disabled state. change btn-dark to btn-secondary and copy the background-color style.